src/main/java/seedu/address/model/person/Person.java

```diff
@@ -72,7 +73,15 @@ public String getPersonType() {
     public void updateCurrentStatus() throws IllegalValueException {
     }
 
-    public String getStatus() {
+    public void updateCurrentStatusToReflectInterview(Model model) {}
+
+    public void updateCurrentStatusToReflectInterview(Model model, Person applicantScheduled) {
+    }
+
+    public void revertCurrentStatus(Model model) {
+    }
+
+    public String getCurrentStatus() {
         return "";
     }
```

Comment:
With the addition of more of such methods, is it ever feasible changing this class to abstract instead and have the respective subclasses override them?